zipkin-server/README.md

```diff
@@ -100,26 +118,50 @@ counter.zipkin_collector.spans_dropped.$transport | cumulative spans dropped; re
 gauge.zipkin_collector.message_spans.$transport | last count of spans in a message
 gauge.zipkin_collector.message_bytes.$transport | last count of bytes in a message
 
-## Self-Tracing
-Self tracing exists to help troubleshoot performance of the zipkin-server. Production deployments
-who enable self-tracing should lower the sample rate from 1.0 (100%) to a much smaller rate, like
-0.001 (0.1% or 1 out of 1000).
-
-When Brave dependencies are in the classpath, and `zipkin.self-tracing.enabled=true`,
-Zipkin will self-trace calls to the api.
+## Configuration
+We support ENV variable configuration, such as `STORAGE_TYPE=cassandra3`, as they are familiar to
+administrators and easy to use in runtime environments such as Docker.
 
-[yaml configuration](src/main/resources/zipkin-server-shared.yml) binds the following environment variables to spring properties:
-
-Variable | Property | Description
---- | --- | ---
-SELF_TRACING_ENABLED | zipkin.self-tracing.enabled | Set to true to enable self-tracing. Defaults to false
-SELF_TRACING_SAMPLE_RATE`: Percentage of self-traces to retain, defaults to always sample (1.0).
-SELF_TRACING_FLUSH_INTERVAL | zipkin.self-tracing.flush-interval | Interval in seconds to flush self-tracing data to storage. Defaults to 1
+Here are the top-level configuration of Zipkin:
+* `QUERY_PORT`: Listen port for the http api and web ui; Defaults to 9411
+* `QUERY_ENABLED`: `false` disables the query api and UI assets. Search
```

Comment:
I don't understand what this sentence is saying, Search may be disabled for the storage backend. Is it possible to rephrase it?

Reply:
this content was just moved up. I can address this now, if you think it is relevant to..

Reply:
agree it is awkward.. I'll do this here as it shouldn't be too much chatter to find good text.